translations/base-tr.yaml

```diff
@@ -622,10 +625,13 @@ storyRewards:
             seç ve <strong>türler arası geçiş yapmak için 'T' tuşuna
             bas</strong>!
     reward_miner_chainable:
-        title: Zincirleme Üretİm
-        desc: "<strong>Zincirleme Üreticiyi</strong> açtın! <strong> Çıkardığı eşyayı önündeki
-            üreticiye aktarabilir</strong> böylelikle madenleri daha efektif kullana bilirsin!<br><br>
-            NOT: Eski üretici Zincirleme Üretici ile değiştirildi!"
+        title: Zincirleme Çıkartıcı
+
+        desc: " <strong>zincirleme çıkarıcıyı</strong>açtın! bununla
+            <strong>kaynakalrını</strong> diğer çıkarıcılarla paylaşıp 
+            daha verimli bir şekilde çıkartabilirsin!<br><br> not: Eskilerini 
+            yenileri ile değiştirdim!"
+
```

Comment:
- Syntax error and word misspelling. 
- Title translation ("Zincirleme Çıkartıcı") is okay but not united with the other "extractor" translations. If this will be used, then other translations of "miner" should be converted into this one